apps/dashboard/src/components/pay/PayAnalytics/hooks/usePayNewCustomers.ts

```diff
@@ -30,7 +31,8 @@ export function usePayNewCustomers(options: {
   to: Date;
   intervalType: "day" | "week";
 }) {
-  const { user, isLoggedIn } = useLoggedInUser();
+  const { user } = useLoggedInUser();
+  const accessToken = useAccessToken();
 
   return useQuery(
     ["usePayNewCustomers", user?.address, options],
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line 41]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Shouldn't this also go through a proxy?<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/3607?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
does now :)